blog/2025/rippled-2.6.0.md

```diff
@@ -12,6 +12,9 @@ markdown:
         hide: true
 ---
 # Introducing XRP Ledger version 2.6.0
+{% admonition type="warning" name="Warning" %}
```

Comment:
Redocly's `type="warning"` is the yellow one, but the title "Warning" (in our old 4-tier system) corresponds to the red one. Use `danger` instead for the red one.

```suggestion
{% admonition type="danger" name="Warning" %}
```